game-core/src/main/java/games/strategy/triplea/ui/DicePanel.java

```diff
@@ -44,15 +46,22 @@ public void setDiceRoll(final DiceRoll diceRoll) {
     SwingAction.invokeNowOrLater(() -> {
       removeAll();
       for (int i = 1; i <= data.getDiceSides(); i++) {
-        final List<Die> dice = diceRoll.getRolls(i);
+        final int index = i;
```

Comment:
It's somewhat weird that we loop over all possible values to extract individual values from all available rolls, but whatever.

Reply:
That's my reaction to a lot of TripleA code too : )
On the plus side our efforts to clean things up are getting very noticeable.